CONTRIBUTING.md

```diff
@@ -180,6 +180,56 @@ See [Translating](#translating) for more details.
 
 There is a dedicated [`apache-superset` tag](https://stackoverflow.com/questions/tagged/apache-superset) on [StackOverflow](https://stackoverflow.com/). Please use it when asking questions.
 
+## Types of Contributors
+
+Following Apache Software Foundation's project governance, Apache Superset has the following types of contributors
+
+### PMC Member
+
+A Project Management Committee (PMC) member is a person who has been elected by the PMC to help manage the project.
```

Comment:
@dpgaspar there are a number of other places where unnecessary newlines are added such as lines 190–192, 198–199, 205–206, etc.